src/explorer.ts

```diff
@@ -665,6 +693,17 @@ export class OpenShiftExplorer implements TreeDataProvider<ExplorerItem>, Dispos
         return await Oc.Instance.getKubernetesObjects('pods', undefined, element.metadata.name, this.executionContext);
     }
 
+    public async getPipelineTasks(element: KubernetesObject | OpenShiftObject): Promise<PipelineTasks[]> {
+        const namespace: string = await Oc.Instance.getActiveProject();
+        const collections: OtherObject[] = await Oc.Instance.getKubernetesObjects(element.kind, namespace, undefined, this.executionContext);
+        const taskNames: PipelineTasks[] = [];
+        const tasks = collections[0].spec.tasks;
```

Comment:
On clean (just created, not having any deployments) Sandbox account I'm receiving the following error:
```
Cannot read properties of undefined (reading 'spec')
```
That's probably happening while going through this line of code